### ISSUE TITLE
1.16.1 Make compatible with FE energy

### ISSUE THREAD

**Issue by ClaudiusMinimus** [gigaherz/Ender-Rift#45]
Could you make your mod compatible with FE energy?

**Comment by gigaherz** [comment]
Uhm, the mod uses FE energy exclusively. I just call it RF because it's the same numbers, 1RF = 1FE

**Comment by ClaudiusMinimus** [comment]
That is true, but mods that specify FE will not transfer power to your mod. For example [Industrial Reborn](https://www.curseforge.com/minecraft/mc-mods/industrial-reborn) generators will not transfer power and their cables will not connect to the Ender-Rift

**Comment by gigaherz** [comment]
Maybe there's a bug, but I didn't change anything energy-related  between 1.15 and 1.16.
Are you connecting it right? The energy goes to the driver block, and any of the corners of the structure, not the rift itself.

**Comment by gigaherz** [comment]
I took another look, Industrial Reborn does not appear to use FE at all. In fact in [the issue you wrote](https://github.com/Maciej916/Industrial-Reborn/issues/8), the author says they have no intention to make their mod compatible with others, so that sounds to me like they *don't* use FE at all, meaning there's no way it would have worked.

I tested with a little mod called [Tesseract](https://www.curseforge.com/minecraft/mc-mods/tesseract), and it was perfectly able to accept and send power. So yeah, closing as it works as intended.
